processing/src/main/java/io/druid/query/groupby/epinephelinae/GroupByMergingQueryRunnerV2.java

```diff
@@ -119,12 +119,13 @@ public GroupByMergingQueryRunnerV2(
         CTX_KEY_MERGE_RUNNERS_USING_CHAINED_EXECUTION,
         false
     );
-    final GroupByQuery queryForRunners = query.withOverriddenContext(
-        ImmutableMap.<String, Object>of(CTX_KEY_MERGE_RUNNERS_USING_CHAINED_EXECUTION, true)
+    final QueryPlus<Row> queryPlusForRunners = queryPlus.withQuery(
+        query.withOverriddenContext(ImmutableMap.<String, Object>of(CTX_KEY_MERGE_RUNNERS_USING_CHAINED_EXECUTION, true))
     );
 
     if (QueryContexts.isBySegment(query) || forceChainedExecution) {
-      return new ChainedExecutionQueryRunner(exec, queryWatcher, queryables).run(query, responseContext);
+      // TODO why query, not queryForRunners?
```

Comment:
Ah, this should be queryForRunners. I think it won't have an effect in practice (since in this block we're already at the lowest level of the merge -- see comment above) so it's not a "bug". But nevertheless, queryForRunners would make more sense here.